dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/Run.java

```diff
@@ -342,6 +342,7 @@ private int run() throws Exception {
         if (dep == null) {
             dep = dependencies;
         } else if (dependencies != null && !dependencies.equals(dep)) {
+            dep = dep.substring(0, dep.lastIndexOf(","));
```

Comment:
What if there is no comma in dep, then indexOf returns -1 and substring will fail. I think we need to check if dep ends with comma we can avoid adding the comma